packages/mgt-components/src/graph/graph.userWithPhoto.ts

```diff
@@ -26,14 +26,20 @@ import { schemas } from './cacheStores';
  * @returns {(Promise<IDynamicPerson>)}
  * @memberof Graph
  */
-export async function getUserWithPhoto(graph: IGraph, userId?: string): Promise<IDynamicPerson> {
+export async function getUserWithPhoto(
+  graph: IGraph,
+  userId?: string,
+  requestedProps?: string[]
+): Promise<IDynamicPerson> {
   let photo = null;
   let user: IDynamicPerson = null;
 
   let cachedPhoto: CachePhoto;
   let cachedUser: CacheUser;
 
-  const resource = userId ? `users/${userId}` : 'me';
+  const propsResource = requestedProps ? `?$select=${requestedProps.toString()}` : '';
+
+  const resource = userId ? `users/${userId}/` + propsResource : 'me';
```

Comment:
make sure to also add the props for the `me` resource

```suggestion
  let resource = userId ? `users/${userId}/` : 'me';
  resource += requestedProps ? `?$select=${requestedProps.toString()}` : '';
```

Reply:
Works well, thanks for the suggestion:

https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/913/commits/f0bbd9e5f1aecffe4c02162fd354979b48354d5d#diff-c43bc0d7ce59f469563d82ee126c8109778544a998ec61d489d8a9d8c589c7cbR40